tests/test_web_functional.py

```diff
@@ -181,6 +181,33 @@ def go():
 
         self.loop.run_until_complete(go())
 
+    def test_files_upload_with_same_key(self):
+        @asyncio.coroutine
+        def handler(request):
+            data = yield from request.post()
+            files = data.getall('file')
+            for _file in files:
+                self.assertEqual(_file.file.closed, False)
```

Comment:
I'd like check for file content for example.
Say, on first cycle iteration it should be `b'binary data 1'` and for second pass `b'binary data 2'`